extensions/amp-megaphone/validator-amp-megaphone.protoascii

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-megaphone
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-megaphone"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-megaphone> for playlists
+  html_format: AMP
+  tag_name: "AMP-MEGAPHONE"
+  requires_extension: "amp-megaphone"
+  spec_name: "amp-megaphone [data-playlist]"
+  attrs: {
+    name: "data-playlist"
+    mandatory: true
+    value_regex: "[A-Za-z0-9]+"
+    dispatch_key: NAME_DISPATCH
+  }
+  attrs: {
```

Comment:
Should create an attribute list to share these common attributes. For an example, see how amp-carousel does this:

https://github.com/ampproject/amphtml/blob/4b2734bd349bd39b72f3de5a619d4cfe8083dd52/extensions/amp-carousel/validator-amp-carousel.protoascii#L33-L39

https://github.com/ampproject/amphtml/blob/4b2734bd349bd39b72f3de5a619d4cfe8083dd52/extensions/amp-carousel/validator-amp-carousel.protoascii#L75-L84